docs/application/tizen-studio-0/setup/test.md

```diff
@@ -0,0 +1,135 @@
+# Grid layout(Tizen 6.0 or late)
+
+> **Note**
+>
+> If you are using GridLayout on Tizen 5.5, Please refer to [Grid Layout(Tizen 5.5)](#gridlayout-tizen5.5) instead.
+
+`GridLayout` is a grid box for the two dimentional layout. It constraints the x and y position, width and height of the child actors.
+
+Positioning children in a grid form, the size of each column and row is set to the largest size of cells belonging to their axis.
+
+If the `Column` or the `row` Property is not specified on the child, it is automtically assigned left to right or top to bottom depends on the `GridOrientation` Property.
+
+
+There are two types of properties. One is the properties for the grid layout and Another one is the properties that affect the specified child:
```

Comment:
Replace "Another one" with "the other" in the sentence.

Also change "grid layout" to "GridLayout"

Reply:
Here the ist sentence is incomplete, we can write: 

There are two types of properties that affect the `GridLayout`. One is the properties for the grid layout and the other is the properties that affect the specified child.